src/database/config.js

```diff
@@ -1,3 +1,7 @@
+import { config } from 'dotenv';
+
+config();
+
 module.exports = {
```

Comment:
we can have `export {}` if we're trying to use es6

Reply:
nice one thanks